src/app/team/team-team-matches/team-team-matches.component.html

```diff
@@ -12,7 +12,7 @@
       <app-team-team-match-card *ngFor="let teamTeamMatch of teamTeamMatches" [teamTeamMatch]="teamTeamMatch"> </app-team-team-match-card>
    </div>
 </div>
-<div *ngIf="teamTeamMatches">
-   <small class="pull-right text-muted"> *прогнози гравців відображаються після початку другого тайму матчу. </small>
-   <small class="pull-right text-muted"> **заблоковані матчі відображаються після добавлення результату. </small>
+<div *ngIf="teamTeamMatches" class="mt-2gitt">
```

Comment:
Is `mt-2gitt` a typo?
```suggestion
<div *ngIf="teamTeamMatches" class="mt-2">
```